src/i18n.js

```diff
@@ -2,58 +2,58 @@ import { createI18n } from 'vue-i18n'
+    'home.transferOngoing': 'Transfer Pending...',
+    'home.transferSuccess': 'Transfer Complete!',
     'home.transfer': 'Transfer',
     'home.gasFee': 'Gas Fee: ',
-    'home.transferTime': 'Time to transfer: {0}',
-    'home.exchangeTime': 'Time to exchange: {0}',
-    'home.lastTransaction': 'Last transaction:',
+    'home.transferTime': 'Time to Transfer to EOS Account: {0}',
+    'home.exchangeTime': 'Time to Trasfer to CEX: {0}',
+    'home.lastTransaction': 'Last Transaction:',
     'home.eosTx': 'Corresponding EOS tx:',
     'home.eosTxPending': 'Querying...',
     'home.eosTxError': 'Failed to locate EOS tx',
-    'home.copyEosTx': 'Copy EOS Tx ID',
-    'home.viewEosTx': 'View on EOS explorer',
-    'home.copyEvmTx': 'Copy EVM Tx ID',
-    'home.viewEvmTx': 'View on EVM explorer',
+    'home.copyEosTx': 'Copy EOS tx ID',
+    'home.viewEosTx': 'View on EOS Block Explorer',
+    'home.copyEvmTx': 'Copy EVM tx ID',
+    'home.viewEvmTx': 'View on EOS EVM Block Explorer',
     'home.copy': 'Copy',
-    'home.addrCopied' : 'Address copied',
+    'home.addrCopied' : 'Address Copied',
     'home.bridgeFee': 'Bridge Fee:',
-    'home.depositDesc': 'Send the transaction and it\'s done!',
-    'home.depositMemoLabel': 'Transfer memo / Destination Tag',
-    'home.addressTitle': 'Please use the destination EVM address:',
-    'home.connectEvmWallet': 'Connect EVM wallet',
-    'home.addressDesc': '{0} to display your address.',
+    'home.depositDesc': 'Transfers to EOS EVM Addresses complete instantly',
+    'home.depositMemoLabel': 'Memo / Destination Tag',
+    'home.addressTitle': 'Please use the destination EOS EVM Address:',
+    'home.connectEvmWallet': 'Connect EOS EVM Wallet',
+    'home.addressDesc': '{0} to display your Address.',
 
     'home.netWarning.testnet': 'Warning! This is the TESTNET.',
     'home.netWarning.mainnet': 'Warning! This is the MAINNET.',
-    'home.netWarning.desc': 'Please confirm that your wallet is connected to the correct network before making a transfer. Using the wrong network may result in potential loss of assets.',
-    'home.eos2evmDesc.p1': 'Please use a wallet that supports the EOS Network, such as Anchor, Wombat, Tokenpocket or a centralized exchange such as Binance, Coinbase, etc.',
-    'home.eos2evmDesc.p2': 'To transfer funds to the following EOS contract address, please fill in the EVM destination address in the memo to complete the deposit to EVM.',
+    'home.netWarning.desc': 'Please confirm that your wallet is connected to the correct network before making a transfer. Using the wrong network may result in loss of tokens.',
+    'home.eos2evmDesc.p1': 'Please use a wallet that supports the EOS Network, such as Anchor, Wombat, TokenPocket or a centralized exchange such as Binance, Coinbase, etc.',
+    'home.eos2evmDesc.p2': 'To transfer funds to the following EOS Contract Address, please fill in the Destination EOS EVM Address in the Memo to complete the deposit to EOS EVM.',
```

Comment:
The capitalization of Memo here intended? I ask because the word "memo" is used in the `memoTooltip` without the first letter being uppercase.